index.html

```diff
@@ -518,6 +518,174 @@ <h2>Thank you!!</h2>
             </section>
 
 
+           <!-- Lesson 2 -->
+            <section>
+                <h1>Introduction to Python</h1>
+                <h3>Lesson 2</h3>
+                <p>
+                    <small>Closely following <a href="https://automatetheboringstuff.com/">Automate the boring stuff with Python</a></small>
+                </p>
+            </section>
+
+             <!-- Booleans -->
+            <section title="Booleans">
+                <section>
+                    <h2>Booleans</h2>
+                    <p> Boolean values </p>
+                    <br>
+                    <small>The Boolean data type has only two values:</small>
+                    <br>
+                    <small>True and False</small>
+                    <small>Boolean is capitalized because the data type is named after mathematician George Boole</small>
+                </section>
+
+                <section>
+                    <h2>Booleans</h2>
+                    <br>
+                    <small>Like any other value, Boolean values are used in expressions and can be stored in variables.
+                        If you don’t use the proper case or you try to use True and False for variable names,
+                        Python will give you an error message.
+                        <br><br>
+                        1. Type the following into your terminal
+                    </small>
+                    <pre><code class="python" data-trim contenteditable>
+                        >>> spam = True #then press enter
+                        >>> spam #if you press enter again what do you think you'll see?
+                    </code></pre>
+                </section>
+
+                <section>
+                    <h2>Booleans</h2>
+                    <br>
+                    <small>
+                        Answer: Because a boolean can be a variable, just like a number or a string it returned True
+                    </small>
+                    <pre><code class="python" data-trim contenteditable>
+                        >>> spam = True
+                        >>> spam
+                        True
+                    </code></pre>
+
+                    <small>
+                        Some more questions to try in your terminal:
+                        <br><br>
+                        2. What happens if you used the wrong case for True?
```

Comment:
rewording needed, like
`What happens if you use <code>true</code> instead of <code>True</True>? Try it out!`

Something like this